javascript/javascript1/week2/event.js

```diff
@@ -0,0 +1,8 @@
+function getEventWeekdag(eventDay){
+    const weekdays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]; 
+    const day= new Date();
+    let todaysDay= dag.getdag();
+    let weekdayOfEvent = todaysDay + eventDay;
+}
+
+console.log(getEventWeekday(9));
```

Comment:
OBS! 😊
There are some things to reconsider in this code. You have a really good starting point and a good logical approach.

First I would suggest to test the code. What output do you get in the console, when you run the code? 😊
- Be aware of "day" vs. "dag" in your code. 

There is a hint in the assignment description that will help you handle scenarios in which you need to calculate you way around the weekend. 
- _Hint: use remainder operator, array indexes and investigate new Date in js_

Don't hesitate to reach out to me on Slack if you would like to talk it through more thoroughly. My slack name is 'Marie Louise Meyer' 😊